mmtrack/datasets/tao_dataset.py

```diff
@@ -0,0 +1,266 @@
+                # (LVIS v1 will fix this naming issue)
+                info['filename'] = info['file_name'][-16:]
+            else:
+                info['filename'] = info['file_name']
+            data_infos.append(info)
+        return data_infos
+
+    def load_tao_anns(self, ann_file):
+        """Load annotation from COCOVID style annotation file.
+
+        Args:
+            ann_file (str): Path of annotation file.
+
+        Returns:
+            list[dict]: Annotation info from COCOVID api.
+        """
+        self.coco = CocoVID(ann_file)
+        self.cat_ids = self.coco.get_cat_ids(cat_names=self.CLASSES)
+        self.cat2label = {cat_id: i for i, cat_id in enumerate(self.cat_ids)}
+
+        data_infos = []
+        self.vid_ids = self.coco.get_vid_ids()
+        self.img_ids = []
+        for vid_id in self.vid_ids:
+            img_ids = self.coco.get_img_ids_from_vid(vid_id)
+            if self.key_img_sampler is not None:
+                img_ids = self.key_img_sampling(img_ids,
+                                                **self.key_img_sampler)
+            self.img_ids.extend(img_ids)
+            for img_id in img_ids:
+                info = self.coco.load_imgs([img_id])[0]
+                if info['file_name'].startswith('COCO'):
+                    # Convert form the COCO 2014 file naming convention of
+                    # COCO_[train/val/test]2014_000000000000.jpg to the 2017
+                    # naming convention of 000000000000.jpg
+                    # (LVIS v1 will fix this naming issue)
+                    info['filename'] = info['file_name'][-16:]
+                else:
+                    info['filename'] = info['file_name']
+                data_infos.append(info)
+        return data_infos
+
+    def _track2json(self, results):
+        """Convert tracking results to TAO json style."""
+        ids = [i for i, _ in enumerate(self.data_infos) if _['frame_id'] == 0]
+        num_vids = len(ids)
+        ids.append(len(self.data_infos))
+        results = [results[ids[i]:ids[i + 1]] for i in range(num_vids)]
+        img_infos = [
+            self.data_infos[ids[i]:ids[i + 1]] for i in range(num_vids)
+        ]
+
+        json_results = []
+        max_track_id = 0
+        for _img_infos, _results in zip(img_infos, results):
+            track_ids = []
+            for img_info, result in zip(_img_infos, _results):
+                img_id = img_info['id']
+                for label in range(len(result)):
+                    bboxes = result[label]
+                    for i in range(bboxes.shape[0]):
+                        data = dict(
+                            image_id=img_id,
+                            bbox=self.xyxy2xywh(bboxes[i, 1:]),
+                            score=float(bboxes[i][-1]),
+                            video_id=img_info['video_id'],
+                            category_id=label
+                            if len(results) == 1230 else self.cat_ids[label],
```

Comment:
add comment to explain that `1230` denotes the number of categories in LVIS dataset.